heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is used to derive a cluster status based
+    # on the status okay or down status for its members.
```

Comment:
See https://github.com/tcpcloud/salt-formula-galera/pull/21/files for Galera/MySQL.